### ISSUE TITLE
[REC] Havoc Opener

### ISSUE THREAD

**Issue by yokaii4** [Hekili/hekili#3788]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Spec

DH - Havoc

### Describe the Issue

After today's update, Havoc opener improved but it's not working properly yet. During prepull, immolation aura gets recommended after sigil of flame and felblade gets recommended before The Hunt until we enter combat. Finally, after the first vengeful Retreat the recommendation is a chaos strike, and not Metamorphosis as it should be

### How to Reproduce

1. Play as Havoc on an ST dummy
2. execute the rotation by using FS raid build

### Snapshot (Link)

prepull: https://pastebin.com/NheXM1uM  felblade rec: https://pastebin.com/aYYqvacj Vengeful Retreat section: https://pastebin.com/7WtK8Ar6  https://pastebin.com/SN2eYDQn

### Raidbots Sim Report (Link)

https://www.raidbots.com/simbot/report/fsJmDtE2WYmVMbs89cGZTJ

### Contact Information

yokaii__

**Comment by yokaii4** [comment]
NOTE: there's also another issue after the first Death Sweep where, before using vengeful retreat and meta, we must cast a Chaos Strike to not waste a Demonsurge, you can also notice that the proc is up. Snapshot: https://pastebin.com/L32CK4u8

**Comment by yokaii4** [comment]
NOTE 2: I have also noticed that the addon struggles to manage the Fury and at some point, it forgets to recommend abilities like chaos blades, immolation aura, or sigil of flame even when they are off cd https://pastebin.com/TfDRWviq

EDIT: another situation where I was struggling on Fury and the recommendations got stuck https://pastebin.com/P3LuSRqn https://pastebin.com/LUcTnZgY

**Comment by yokaii4** [comment]
NOTE 3: For this one I can't catch a snapshot for now because it's hard to reproduce, however sometimes the addon recommends vengeful retreat when Eye Beam is almost ready, making us lose the initiative proc. Secondly, it happened once that instead of waiting Essence Break to be off cd (2-3s left) the addon recommended using death sweep making us lose another dps proc

EDIT: not the same thing, but the addon failed to pair vengeful retreat with meta here by recommending retreat randomly when meta was coming back in 8s https://pastebin.com/XN31VuKi

**Comment by yokaii4** [comment]
NOTE 4: Sometimes even if both EB and Death Sweep are up, the addon recommends to cast DS before EB. Also, there are still some issues sometimes where immo aura, sigil and blade dance don't get recommended even if they are off cd

**Comment by Hekili** [comment]
> During prepull, immolation aura gets recommended after sigil of flame

Yes.

![image](https://github.com/user-attachments/assets/69ce9e06-d12e-4ca2-898e-42a787299689)

> and felblade gets recommended before The Hunt until we enter combat. 

Yes.

![image](https://github.com/user-attachments/assets/614137f7-8ccb-4875-9662-a99d8168e17b)

The conditions for this Felblade in the `opener` action list are:

`felblade,if=(buff.metamorphosis.down|fury<40)&(!talent.a_fire_inside|hero_tree.aldrachi_reaver)&action.felblade.cooldown_react`

I'll bold the portions where criteria are met in your snapshot for the ability to be recommended.

> - this entry's criteria PASSES: ( **buff.metamorphosis.down[true]** | fury.current[55.00] < 40 ) & ( **! talent.a_fire_inside.enabled[false]** | hero_tree.aldrachi_reaver[false] ) & **action.felblade.cooldown_react[true]**`

Both the sim and the addon are using the same criteria; you can see that this entry would disappear if _both_ Metamorphosis were active _and_ your Fury reaches 40+.  If Felblade vanishes under other circumstances, please provide a snapshot that shows this.

> Finally, after the first vengeful Retreat the recommendation is a chaos strike, and not Metamorphosis as it should be

Here, we have a discrepancy to look at:

`vengeful_retreat,use_off_gcd=1,if=(prev_gcd.1.death_sweep&buff.initiative.remains<2&buff.inner_demon.down|buff.initiative.remains<0.5&debuff.initiative_tracker.up&(!talent.inertia|buff.unbound_chaos.down))&((!talent.essence_break&talent.shattered_destiny)|(talent.essence_break&!talent.shattered_destiny)|(!talent.shattered_destiny&!talent.essence_break))|prev_gcd.2.death_sweep&prev_gcd.1.annihilation`

> - this entry's criteria PASSES: ( **prev_gcd[1].death_sweep[true]** & buff.initiative.remains[0.00] < 2 & buff.inner_demon.down[false] | **buff.initiative.remains[0.00] < 0.5 & debuff.initiative_tracker.up[true]** & ( ! talent.inertia.enabled[true] | **buff.unbound_chaos.down[true]** ) ) & ( ( ! talent.essence_break.enabled[true] & talent.shattered_destiny.enabled[false] ) | ( **talent.essence_break.enabled[true] & ! talent.shattered_destiny.enabled[false]** ) | ( ! talent.shattered_destiny.enabled[false] & ! talent.essence_break.enabled[true] ) ) | prev_gcd[2].death_sweep[false] & prev_gcd[1].annihilation[false]

It's *possible* that `debuff.initiative_tracker.up` is wrong, in which case this entry wouldn't show yet; this entry *would* then show after Annihilate` because `prev_gcd[2].death_sweep & prev_gcd[1].annihilation` will flip to true.  I'll look into it.

> NOTE: there's also another issue after the first Death Sweep where, before using vengeful retreat and meta, we must cast a Chaos Strike to not waste a Demonsurge, you can also notice that the proc is up. Snapshot: [pastebin.com/L32CK4u8](https://pastebin.com/L32CK4u8)

This is the same as previous.  The Vengeful Retreat criteria are met, but the Annihilation entry has to happen before Metamorphosis' criteria are met (to avoid wasting the Demonsurge: Annihilation):

`metamorphosis,if=...`

> - this entry's criteria FAILS: ( ( ! talent.initiative.enabled[true] | cooldown.vengeful_retreat.remains[20.00] ) & cooldown.blade_dance.remains[5.03] & ( ( cooldown.death_sweep.remains[5.03] | buff.metamorphosis.up[true] & buff.metamorphosis.remains[2.53] < ( gcd.max[1.00] * 1.5 ) ) & cooldown.eye_beam.remains[35.90] & ( ! talent.essence_break.enabled[true] | debuff.essence_break.up[false] | cooldown.essence_break.true_remains[0.00] > cooldown.essence_break.duration[40.00] - debuff.essence_break.duration[4.00] | talent.shattered_destiny.enabled[false] | hero_tree.felscarred[true] ) & buff.fel_barrage.down[true] & ( raid_event.adds['in'][3600.00] > 40 | ( raid_event.adds.remains[0.00] > 8 | ! talent.fel_barrage.enabled[false] ) & active_enemies[1.00] > 2 ) | ! talent.chaotic_transformation.enabled[true] | fight_remains[98.32] < 30 ) & ( buff.inner_demon.down[false] & ( buff.rending_strike.down[true] | ! talent.restless_hunter.enabled[false] | prev_gcd[1].death_sweep[true] ) ) ) & ( talent.inertia.enabled[true] | talent.essence_break.enabled[true] ) & ( hero_tree.aldrachi_reaver[false] | ( buff.demonsurge_death_sweep.down[true] | buff.metamorphosis.remains[2.53] < gcd.max[1.00] ) & ( `buff.demonsurge_annihilation.down[false]` ) ) & time[6.20] > 15

The Annihilation cast actually has no criteria; it's a filler entry that spends the Demonsurge: Annihilation buff, so the Metamorphosis criteria are met above.

If `initiative_tracker` is the source of the previous issue, it will also solve for this item.

> NOTE 2: I have also noticed that the addon struggles to manage the Fury and at some point, it forgets to recommend abilities like chaos blades, immolation aura, or sigil of flame even when they are off cd [pastebin.com/TfDRWviq](https://pastebin.com/TfDRWviq)

* Chaos Blades is not an ability -- you have Demon Blades talented, so your Fury generation is conservatively predicted from auto-attacks.  You can't use Chaos Strike.
* Immolation Aura is on cooldown for 9.7 seconds and is recommended in your queue as soon as it comes off CD.
* Sigil of Flame is off cooldown and has multiple entries that were ruled out:

> - this entry's criteria FAILS: talent.student_of_suffering.enabled[true] & cooldown.eye_beam.remains[0.00] < gcd.max[1.00] & ( ! talent.inertia.enabled[true] | buff.inertia_trigger.up[false] ) & ( cooldown.essence_break.remains[0.00] < gcd.max[1.00] * 4 | ! talent.essence_break.enabled[true] ) & ( cooldown.metamorphosis.remains[27.27] > 10 | cooldown.blade_dance.remains[4.61] < gcd.max[1.00] * 3 ) & ( ! variable.trinket1_steroids[false] & ! variable.trinket2_steroids[false] | variable.trinket1_steroids[false] & ( trinket.t1.cooldown.remains[0.00] < gcd.max[1.00] * 3 | trinket.t1.cooldown.remains[0.00] > 20 ) | variable.trinket2_steroids[false] & ( trinket.t2.cooldown.remains[0.00] < gcd.max[1.00] * 3 | trinket.t2.cooldown.remains[0.00] > 20 ) )

This failed because `buff.inertia_trigger.up` is false, so I'll check that it's implemented correctly.

> EDIT: another situation where I was struggling on Fury and the recommendations got stuck [pastebin.com/P3LuSRqn](https://pastebin.com/P3LuSRqn) [pastebin.com/LUcTnZgY](https://pastebin.com/LUcTnZgY)

What does "got stuck" mean?  What are you expecting it to recommend differently, and why?  It looks like these recommendations are mainly impacted by Eye Beam coming off CD in 15 seconds and having to conservatively estimate Fury gains from auto-attacks in the meantime.

> NOTE 3: For this one I can't catch a snapshot for now because it's hard to reproduce, however sometimes the addon recommends vengeful retreat when Eye Beam is almost ready, making us lose the initiative proc. Secondly, it happened once that instead of waiting Essence Break to be off cd (2-3s left) the addon recommended using death sweep making us lose another dps proc

Will need snapshot(s).

> EDIT: not the same thing, but the addon failed to pair vengeful retreat with meta here by recommending retreat randomly when meta was coming back in 8s [pastebin.com/XN31VuKi](https://pastebin.com/XN31VuKi)

Would need to snapshot the Vengeful Retreat recommendation; it may be related to `inertia_trigger` or `initiative_tracker` above.

> NOTE 4: Sometimes even if both EB and Death Sweep are up, the addon recommends to cast DS before EB. Also, there are still some issues sometimes where immo aura, sigil and blade dance don't get recommended even if they are off cd

Requires a snapshot.

**Comment by yokaii4** [comment]
@Hekili I will try to get snapshots where needed but as mentioned it's kinda random mid fight so I won't be quick enough to catch it. As for this 

> What does "got stuck" mean? What are you expecting it to recommend differently, and why? It looks like these recommendations are mainly impacted by Eye Beam coming off CD in 15 seconds and having to conservatively estimate Fury gains from auto-attacks in the meantime.

I expect the addon to recommend sigil and immo aura more often especially when we are running low fury. Instead, it is too conservative even if the spells will be available before either an eye beam combo or meta. Essentially it is too conservative because it starts around 10s before those 15s so you have around 5-6 seconds of auto attack only with the recommendations atm

> Chaos Blades is not an ability -- you have Demon Blades talented, so your Fury generation is conservatively predicted from auto-attacks. You can't use Chaos Strike.

Misspelled Chaos Strike in "Chaos Blades" on my first message

Managed to get a snapshot of when I have sigil and immo aura available and low on fury but the system is too conservative (you could immo aura to get instant fury and feel rush later and still proc inertia): https://pastebin.com/1dck2MhH

> Both the sim and the addon are using the same criteria; you can see that this entry would disappear if both Metamorphosis were active and your Fury reaches 40+. If Felblade vanishes under other circumstances, please provide a snapshot that shows this.

like this? https://justpaste.it.com/1ncsltaajt

**Comment by Hekili** [comment]
> I expect the addon to recommend sigil and immo aura more often especially when we are running low fury. Instead, it is too conservative even if the spells will be available before either an eye beam combo or meta. Essentially it is too conservative because it starts around 10s before those 15s so you have around 5-6 seconds of auto attack only with the recommendations atm

It looks like the sim is doing this as well; there is a 38 second gap between Immolation Auras starting at 2:18 unless I'm overlooking something.  You may want to submit a SimC ticket with your Raidbots link.  There's also a 4 second wait inside of that span where, you're right, Immolation Aura could've been used and generated some Fury.

Time | # | Name [List] | Target | Resources | Buffs
-- | -- | -- | -- | -- | --
2:18.799 | b | immolation_aura[meta] | Yokaidh | 170.0/170 100% fury | metamorphosis, furious_gaze, tactical_retreat,  demonsurge_sigil_of_doom, demonsurge_consuming_fire,  demonsurge_abyssal_gaze, demonsurge_demonic, demonsurge_hardcast,  demonsurge(2), egg_sac(5), Ascendance_Haste(10),  flask_of_alchemical_chaos_vers
2:19.871 | c | sigil_of_doom[meta] | Fluffy_Pillow | 170.0/170 100% fury | immolation_aura, metamorphosis, furious_gaze, tactical_retreat,  unbound_chaos, demonsurge_sigil_of_doom, demonsurge_abyssal_gaze,  demonsurge_demonic, demonsurge_hardcast, demonsurge(3), egg_sac(5),  Ascendance_Haste(10), flask_of_alchemical_chaos_vers
2:20.941 | f | abyssal_gaze[meta] | Fluffy_Pillow | 170.0/170 100% fury | immolation_aura, metamorphosis, furious_gaze, unbound_chaos,  student_of_suffering, demonsurge_abyssal_gaze, demonsurge_demonic,  demonsurge_hardcast, demonsurge(4), egg_sac(5), Ascendance_Haste(10),  flask_of_alchemical_chaos_vers
2:22.462 | Z | fel_rush[meta] | Fluffy_Pillow | 144.0/170 85% fury | immolation_aura, metamorphosis, furious_gaze, inner_demon,  unbound_chaos, student_of_suffering, demonsurge_demonic,  demonsurge_hardcast, demonsurge(5), egg_sac(5), Ascendance_Haste(10),  flask_of_alchemical_chaos_vers
2:23.008 | a | essence_break[meta] | Fluffy_Pillow | 170.0/170 100% fury | immolation_aura, metamorphosis, out_of_range, furious_gaze, inertia,  inner_demon, student_of_suffering, demonsurge_demonic,  demonsurge_hardcast, demonsurge(5), egg_sac(5), Ascendance_Haste(10),  flask_of_alchemical_chaos_vers
2:24.081 | e | death_sweep[meta] | Fluffy_Pillow | 170.0/170 100% fury | immolation_aura, metamorphosis, furious_gaze, inertia, inner_demon,  student_of_suffering, demonsurge_demonic, demonsurge_hardcast,  demonsurge(5), egg_sac(5), Ascension_Mastery(10),  flask_of_alchemical_chaos_vers
2:25.180 | g | annihilation[meta] | Fluffy_Pillow | 170.0/170 100% fury | immolation_aura, metamorphosis, furious_gaze, inertia, inner_demon,  chaos_theory, student_of_suffering, demonsurge_demonic,  demonsurge_hardcast, demonsurge(5), egg_sac(6), Ascension_Mastery(10),  flask_of_alchemical_chaos_vers
2:26.262 | A | pick_up_fragment[default] | Fluffy_Pillow | 154.0/170 91% fury | immolation_aura, metamorphosis, furious_gaze, inertia,  student_of_suffering, demonsurge_demonic, demonsurge_hardcast,  demonsurge(5), egg_sac(6), Ascension_Mastery(10),  flask_of_alchemical_chaos_vers
2:26.283 | g | annihilation[meta] | Fluffy_Pillow | 154.0/170 91% fury | immolation_aura, metamorphosis, furious_gaze, inertia,  student_of_suffering, demonsurge_demonic, demonsurge_hardcast,  demonsurge(5), egg_sac(6), Ascension_Mastery(10),  flask_of_alchemical_chaos_vers
2:27.384 | g | annihilation[meta] | Fluffy_Pillow | 170.0/170 100% fury | immolation_aura, metamorphosis, furious_gaze, inertia,  student_of_suffering, demonsurge_demonic, demonsurge_hardcast,  demonsurge(5), egg_sac(6), Ascension_Mastery(10),  flask_of_alchemical_chaos_mastery
2:28.485 | g | annihilation[meta] | Fluffy_Pillow | 170.0/170 100% fury | immolation_aura, metamorphosis, furious_gaze, student_of_suffering,  demonsurge_demonic, demonsurge_hardcast, demonsurge(5), egg_sac(6),  authority_of_radiant_power, Ascension_Mastery(10),  flask_of_alchemical_chaos_mastery
2:29.585 | g | annihilation[meta] | Fluffy_Pillow | 139.0/170 82% fury | metamorphosis, furious_gaze, demonsurge_demonic,  demonsurge_hardcast, demonsurge(5), spiderling, egg_sac(5),  authority_of_radiant_power, Ascension_Mastery(10),  flask_of_alchemical_chaos_mastery
2:30.640 | A | pick_up_fragment[default] | Fluffy_Pillow | 151.0/170 89% fury | metamorphosis, furious_gaze, demonsurge_demonic,  demonsurge_hardcast, demonsurge(5), egg_sac(5),  authority_of_radiant_power, Ascension_Mastery(10),  flask_of_alchemical_chaos_mastery
2:30.687 | F | vengeful_retreat[default] | Fluffy_Pillow | 151.0/170 89% fury | metamorphosis, furious_gaze, demonsurge_demonic,  demonsurge_hardcast, demonsurge(5), egg_sac(5),  authority_of_radiant_power, Ascension_Mastery(10),  flask_of_alchemical_chaos_mastery
2:30.734 | e | death_sweep[meta] | Fluffy_Pillow | 151.0/170 89% fury | metamorphosis, vengeful_retreat_movement, furious_gaze, initiative,  tactical_retreat, demonsurge_demonic, demonsurge_hardcast,  demonsurge(5), egg_sac(5), authority_of_radiant_power,  Ascension_Mastery(10), flask_of_alchemical_chaos_mastery
2:31.836 |   | Waiting | 0.385s | 144.0/170 85% fury | metamorphosis, out_of_range, furious_gaze, initiative,  tactical_retreat, chaos_theory, demonsurge_demonic, demonsurge_hardcast,  demonsurge(5), egg_sac(4), authority_of_radiant_power,  Ascension_Mastery(10), flask_of_alchemical_chaos_mastery
2:32.221 | 9 | auto_attack[default] | Fluffy_Pillow | 144.0/170 85% fury | metamorphosis, furious_gaze, initiative, tactical_retreat,  chaos_theory, demonsurge_demonic, demonsurge_hardcast, demonsurge(5),  egg_sac(4), authority_of_radiant_power, Ascension_Crit(10),  flask_of_alchemical_chaos_mastery
2:32.221 | g | annihilation[meta] | Fluffy_Pillow | 144.0/170 85% fury | metamorphosis, furious_gaze, initiative, tactical_retreat,  chaos_theory, demonsurge_demonic, demonsurge_hardcast, demonsurge(5),  egg_sac(4), authority_of_radiant_power, Ascension_Crit(10),  flask_of_alchemical_chaos_mastery
2:33.322 | g | annihilation[meta] | Fluffy_Pillow | 149.0/170 88% fury | metamorphosis, initiative, tactical_retreat, demonsurge_demonic,  demonsurge_hardcast, demonsurge(5), egg_sac(4),  authority_of_radiant_power, Ascension_Crit(10),  flask_of_alchemical_chaos_mastery
2:34.532 | g | annihilation[meta] | Fluffy_Pillow | 137.0/170 81% fury | metamorphosis, initiative, tactical_retreat, demonsurge_demonic,  demonsurge_hardcast, egg_sac(4), authority_of_radiant_power,  Ascension_Crit(10), flask_of_alchemical_chaos_mastery
2:35.743 | g | annihilation[meta] | Fluffy_Pillow | 161.0/170 95% fury | metamorphosis, tactical_retreat, demonsurge_demonic,  demonsurge_hardcast, egg_sac(4), authority_of_radiant_power,  Ascension_Crit(10), flask_of_alchemical_chaos_mastery
2:36.954 | g | annihilation[meta] | Fluffy_Pillow | 156.0/170 92% fury | metamorphosis, tactical_retreat, demonsurge_demonic,  demonsurge_hardcast, egg_sac(4), authority_of_radiant_power,  Ascension_Crit(10), flask_of_alchemical_chaos_mastery
2:38.165 | e | death_sweep[meta] | Fluffy_Pillow | 124.0/170 73% fury | metamorphosis, tactical_retreat, demonsurge_demonic,  demonsurge_hardcast, spiderling, egg_sac(3), authority_of_radiant_power,  Ascension_Crit(10), flask_of_alchemical_chaos_mastery
2:39.377 | g | annihilation[meta] | Fluffy_Pillow | 112.0/170 66% fury | metamorphosis, tactical_retreat, chaos_theory, demonsurge_demonic,  demonsurge_hardcast, egg_sac(3), authority_of_radiant_power,  Ascension_Crit(10), flask_of_alchemical_chaos_mastery
2:40.438 | A | pick_up_fragment[default] | Fluffy_Pillow | 100.0/170 59% fury | metamorphosis, tactical_retreat, demonsurge_demonic,  demonsurge_hardcast, egg_sac(3), authority_of_radiant_power,  Ascension_Crit(10), flask_of_alchemical_chaos_mastery
2:40.588 | g | annihilation[meta] | Fluffy_Pillow | 100.0/170 59% fury | metamorphosis, tactical_retreat, demonsurge_demonic,  demonsurge_hardcast, egg_sac(3), authority_of_radiant_power,  Ascension_Crit(10), flask_of_alchemical_chaos_mastery
2:41.695 | A | pick_up_fragment[default] | Fluffy_Pillow | 120.0/170 71% fury | metamorphosis, demonsurge_demonic, demonsurge_hardcast, egg_sac(3),  authority_of_radiant_power, Ascension_Crit(10),  flask_of_alchemical_chaos_mastery
2:41.798 | g | annihilation[meta] | Fluffy_Pillow | 120.0/170 71% fury | metamorphosis, demonsurge_demonic, demonsurge_hardcast, egg_sac(3),  authority_of_radiant_power, Ascension_Crit(10),  flask_of_alchemical_chaos_mastery
2:43.008 | W | annihilation[meta] | Fluffy_Pillow | 100.0/170 59% fury | metamorphosis, demonsurge_demonic, demonsurge_hardcast, egg_sac(3),  authority_of_radiant_power, Ascension_Crit(10),  flask_of_alchemical_chaos_mastery
2:44.218 | O | felblade[default] | Fluffy_Pillow | 89.0/170 52% fury | enduring_torment, monster_rising, egg_sac(3), Ascension_Crit(10), flask_of_alchemical_chaos_mastery
2:44.722 | R | chaos_strike[default] | Fluffy_Pillow | 129.0/170 76% fury | enduring_torment, monster_rising, egg_sac(3), Ascension_Crit(10), flask_of_alchemical_chaos_mastery
2:46.106 | N | blade_dance[default] | Fluffy_Pillow | 104.0/170 61% fury | enduring_torment, monster_rising, egg_sac(3), Ascension_Crit(10), flask_of_alchemical_chaos_mastery
2:47.490 | R | chaos_strike[default] | Fluffy_Pillow | 69.0/170 41% fury | chaos_theory, enduring_torment, monster_rising, egg_sac(3),  authority_of_radiant_power, Ascension_Crit(10),  flask_of_alchemical_chaos_mastery
2:48.873 | R | chaos_strike[default] | Fluffy_Pillow | 60.0/170 35% fury | enduring_torment, monster_rising, egg_sac(3), authority_of_radiant_power, Ascendance_Vers(10), flask_of_alchemical_chaos_mastery
2:50.256 |   | Waiting | 0.107s | 20.0/170 12% fury | enduring_torment, monster_rising, egg_sac(3), authority_of_radiant_power, Ascendance_Vers(10), flask_of_alchemical_chaos_mastery
2:50.363 | R | chaos_strike[default] | Fluffy_Pillow | 41.0/170 24% fury | enduring_torment, monster_rising, egg_sac(3), authority_of_radiant_power, Ascendance_Vers(10), flask_of_alchemical_chaos_mastery
2:51.746 |   | Waiting | 4.268s | 1.0/170 1% fury | enduring_torment, monster_rising, egg_sac(3), authority_of_radiant_power, Ascendance_Vers(10), flask_of_alchemical_chaos_mastery
2:56.014 | J | immolation_aura[default] | Yokaidh | 18.0/170 11% fury | enduring_torment, monster_rising, egg_sac(3), authority_of_radiant_power, Ascendance_Vers(10), flask_of_alchemical_chaos_mastery

**Comment by yokaii4** [comment]
> > I expect the addon to recommend sigil and immo aura more often especially when we are running low fury. Instead, it is too conservative even if the spells will be available before either an eye beam combo or meta. Essentially it is too conservative because it starts around 10s before those 15s so you have around 5-6 seconds of auto attack only with the recommendations atm
> 
> It looks like the sim is doing this as well; there is a 38 second gap between Immolation Auras starting at 2:18 unless I'm overlooking something. You may want to submit a SimC ticket with your Raidbots link. There's also a 4 second wait inside of that span where, you're right, Immolation Aura could've been used and generated some Fury.
> 
> Time	#	Name [List]	Target	Resources	Buffs
> 2:18.799	b	immolation_aura[meta]	Yokaidh	170.0/170 100% fury	metamorphosis, furious_gaze, tactical_retreat, demonsurge_sigil_of_doom, demonsurge_consuming_fire, demonsurge_abyssal_gaze, demonsurge_demonic, demonsurge_hardcast, demonsurge(2), egg_sac(5), Ascendance_Haste(10), flask_of_alchemical_chaos_vers
> 2:19.871	c	sigil_of_doom[meta]	Fluffy_Pillow	170.0/170 100% fury	immolation_aura, metamorphosis, furious_gaze, tactical_retreat, unbound_chaos, demonsurge_sigil_of_doom, demonsurge_abyssal_gaze, demonsurge_demonic, demonsurge_hardcast, demonsurge(3), egg_sac(5), Ascendance_Haste(10), flask_of_alchemical_chaos_vers
> 2:20.941	f	abyssal_gaze[meta]	Fluffy_Pillow	170.0/170 100% fury	immolation_aura, metamorphosis, furious_gaze, unbound_chaos, student_of_suffering, demonsurge_abyssal_gaze, demonsurge_demonic, demonsurge_hardcast, demonsurge(4), egg_sac(5), Ascendance_Haste(10), flask_of_alchemical_chaos_vers
> 2:22.462	Z	fel_rush[meta]	Fluffy_Pillow	144.0/170 85% fury	immolation_aura, metamorphosis, furious_gaze, inner_demon, unbound_chaos, student_of_suffering, demonsurge_demonic, demonsurge_hardcast, demonsurge(5), egg_sac(5), Ascendance_Haste(10), flask_of_alchemical_chaos_vers
> 2:23.008	a	essence_break[meta]	Fluffy_Pillow	170.0/170 100% fury	immolation_aura, metamorphosis, out_of_range, furious_gaze, inertia, inner_demon, student_of_suffering, demonsurge_demonic, demonsurge_hardcast, demonsurge(5), egg_sac(5), Ascendance_Haste(10), flask_of_alchemical_chaos_vers
> 2:24.081	e	death_sweep[meta]	Fluffy_Pillow	170.0/170 100% fury	immolation_aura, metamorphosis, furious_gaze, inertia, inner_demon, student_of_suffering, demonsurge_demonic, demonsurge_hardcast, demonsurge(5), egg_sac(5), Ascension_Mastery(10), flask_of_alchemical_chaos_vers
> 2:25.180	g	annihilation[meta]	Fluffy_Pillow	170.0/170 100% fury	immolation_aura, metamorphosis, furious_gaze, inertia, inner_demon, chaos_theory, student_of_suffering, demonsurge_demonic, demonsurge_hardcast, demonsurge(5), egg_sac(6), Ascension_Mastery(10), flask_of_alchemical_chaos_vers
> 2:26.262	A	pick_up_fragment[default]	Fluffy_Pillow	154.0/170 91% fury	immolation_aura, metamorphosis, furious_gaze, inertia, student_of_suffering, demonsurge_demonic, demonsurge_hardcast, demonsurge(5), egg_sac(6), Ascension_Mastery(10), flask_of_alchemical_chaos_vers
> 2:26.283	g	annihilation[meta]	Fluffy_Pillow	154.0/170 91% fury	immolation_aura, metamorphosis, furious_gaze, inertia, student_of_suffering, demonsurge_demonic, demonsurge_hardcast, demonsurge(5), egg_sac(6), Ascension_Mastery(10), flask_of_alchemical_chaos_vers
> 2:27.384	g	annihilation[meta]	Fluffy_Pillow	170.0/170 100% fury	immolation_aura, metamorphosis, furious_gaze, inertia, student_of_suffering, demonsurge_demonic, demonsurge_hardcast, demonsurge(5), egg_sac(6), Ascension_Mastery(10), flask_of_alchemical_chaos_mastery
> 2:28.485	g	annihilation[meta]	Fluffy_Pillow	170.0/170 100% fury	immolation_aura, metamorphosis, furious_gaze, student_of_suffering, demonsurge_demonic, demonsurge_hardcast, demonsurge(5), egg_sac(6), authority_of_radiant_power, Ascension_Mastery(10), flask_of_alchemical_chaos_mastery
> 2:29.585	g	annihilation[meta]	Fluffy_Pillow	139.0/170 82% fury	metamorphosis, furious_gaze, demonsurge_demonic, demonsurge_hardcast, demonsurge(5), spiderling, egg_sac(5), authority_of_radiant_power, Ascension_Mastery(10), flask_of_alchemical_chaos_mastery
> 2:30.640	A	pick_up_fragment[default]	Fluffy_Pillow	151.0/170 89% fury	metamorphosis, furious_gaze, demonsurge_demonic, demonsurge_hardcast, demonsurge(5), egg_sac(5), authority_of_radiant_power, Ascension_Mastery(10), flask_of_alchemical_chaos_mastery
> 2:30.687	F	vengeful_retreat[default]	Fluffy_Pillow	151.0/170 89% fury	metamorphosis, furious_gaze, demonsurge_demonic, demonsurge_hardcast, demonsurge(5), egg_sac(5), authority_of_radiant_power, Ascension_Mastery(10), flask_of_alchemical_chaos_mastery
> 2:30.734	e	death_sweep[meta]	Fluffy_Pillow	151.0/170 89% fury	metamorphosis, vengeful_retreat_movement, furious_gaze, initiative, tactical_retreat, demonsurge_demonic, demonsurge_hardcast, demonsurge(5), egg_sac(5), authority_of_radiant_power, Ascension_Mastery(10), flask_of_alchemical_chaos_mastery
> 2:31.836	 	Waiting	0.385s	144.0/170 85% fury	metamorphosis, out_of_range, furious_gaze, initiative, tactical_retreat, chaos_theory, demonsurge_demonic, demonsurge_hardcast, demonsurge(5), egg_sac(4), authority_of_radiant_power, Ascension_Mastery(10), flask_of_alchemical_chaos_mastery
> 2:32.221	9	auto_attack[default]	Fluffy_Pillow	144.0/170 85% fury	metamorphosis, furious_gaze, initiative, tactical_retreat, chaos_theory, demonsurge_demonic, demonsurge_hardcast, demonsurge(5), egg_sac(4), authority_of_radiant_power, Ascension_Crit(10), flask_of_alchemical_chaos_mastery
> 2:32.221	g	annihilation[meta]	Fluffy_Pillow	144.0/170 85% fury	metamorphosis, furious_gaze, initiative, tactical_retreat, chaos_theory, demonsurge_demonic, demonsurge_hardcast, demonsurge(5), egg_sac(4), authority_of_radiant_power, Ascension_Crit(10), flask_of_alchemical_chaos_mastery
> 2:33.322	g	annihilation[meta]	Fluffy_Pillow	149.0/170 88% fury	metamorphosis, initiative, tactical_retreat, demonsurge_demonic, demonsurge_hardcast, demonsurge(5), egg_sac(4), authority_of_radiant_power, Ascension_Crit(10), flask_of_alchemical_chaos_mastery
> 2:34.532	g	annihilation[meta]	Fluffy_Pillow	137.0/170 81% fury	metamorphosis, initiative, tactical_retreat, demonsurge_demonic, demonsurge_hardcast, egg_sac(4), authority_of_radiant_power, Ascension_Crit(10), flask_of_alchemical_chaos_mastery
> 2:35.743	g	annihilation[meta]	Fluffy_Pillow	161.0/170 95% fury	metamorphosis, tactical_retreat, demonsurge_demonic, demonsurge_hardcast, egg_sac(4), authority_of_radiant_power, Ascension_Crit(10), flask_of_alchemical_chaos_mastery
> 2:36.954	g	annihilation[meta]	Fluffy_Pillow	156.0/170 92% fury	metamorphosis, tactical_retreat, demonsurge_demonic, demonsurge_hardcast, egg_sac(4), authority_of_radiant_power, Ascension_Crit(10), flask_of_alchemical_chaos_mastery
> 2:38.165	e	death_sweep[meta]	Fluffy_Pillow	124.0/170 73% fury	metamorphosis, tactical_retreat, demonsurge_demonic, demonsurge_hardcast, spiderling, egg_sac(3), authority_of_radiant_power, Ascension_Crit(10), flask_of_alchemical_chaos_mastery
> 2:39.377	g	annihilation[meta]	Fluffy_Pillow	112.0/170 66% fury	metamorphosis, tactical_retreat, chaos_theory, demonsurge_demonic, demonsurge_hardcast, egg_sac(3), authority_of_radiant_power, Ascension_Crit(10), flask_of_alchemical_chaos_mastery
> 2:40.438	A	pick_up_fragment[default]	Fluffy_Pillow	100.0/170 59% fury	metamorphosis, tactical_retreat, demonsurge_demonic, demonsurge_hardcast, egg_sac(3), authority_of_radiant_power, Ascension_Crit(10), flask_of_alchemical_chaos_mastery
> 2:40.588	g	annihilation[meta]	Fluffy_Pillow	100.0/170 59% fury	metamorphosis, tactical_retreat, demonsurge_demonic, demonsurge_hardcast, egg_sac(3), authority_of_radiant_power, Ascension_Crit(10), flask_of_alchemical_chaos_mastery
> 2:41.695	A	pick_up_fragment[default]	Fluffy_Pillow	120.0/170 71% fury	metamorphosis, demonsurge_demonic, demonsurge_hardcast, egg_sac(3), authority_of_radiant_power, Ascension_Crit(10), flask_of_alchemical_chaos_mastery
> 2:41.798	g	annihilation[meta]	Fluffy_Pillow	120.0/170 71% fury	metamorphosis, demonsurge_demonic, demonsurge_hardcast, egg_sac(3), authority_of_radiant_power, Ascension_Crit(10), flask_of_alchemical_chaos_mastery
> 2:43.008	W	annihilation[meta]	Fluffy_Pillow	100.0/170 59% fury	metamorphosis, demonsurge_demonic, demonsurge_hardcast, egg_sac(3), authority_of_radiant_power, Ascension_Crit(10), flask_of_alchemical_chaos_mastery
> 2:44.218	O	felblade[default]	Fluffy_Pillow	89.0/170 52% fury	enduring_torment, monster_rising, egg_sac(3), Ascension_Crit(10), flask_of_alchemical_chaos_mastery
> 2:44.722	R	chaos_strike[default]	Fluffy_Pillow	129.0/170 76% fury	enduring_torment, monster_rising, egg_sac(3), Ascension_Crit(10), flask_of_alchemical_chaos_mastery
> 2:46.106	N	blade_dance[default]	Fluffy_Pillow	104.0/170 61% fury	enduring_torment, monster_rising, egg_sac(3), Ascension_Crit(10), flask_of_alchemical_chaos_mastery
> 2:47.490	R	chaos_strike[default]	Fluffy_Pillow	69.0/170 41% fury	chaos_theory, enduring_torment, monster_rising, egg_sac(3), authority_of_radiant_power, Ascension_Crit(10), flask_of_alchemical_chaos_mastery
> 2:48.873	R	chaos_strike[default]	Fluffy_Pillow	60.0/170 35% fury	enduring_torment, monster_rising, egg_sac(3), authority_of_radiant_power, Ascendance_Vers(10), flask_of_alchemical_chaos_mastery
> 2:50.256	 	Waiting	0.107s	20.0/170 12% fury	enduring_torment, monster_rising, egg_sac(3), authority_of_radiant_power, Ascendance_Vers(10), flask_of_alchemical_chaos_mastery
> 2:50.363	R	chaos_strike[default]	Fluffy_Pillow	41.0/170 24% fury	enduring_torment, monster_rising, egg_sac(3), authority_of_radiant_power, Ascendance_Vers(10), flask_of_alchemical_chaos_mastery
> 2:51.746	 	Waiting	4.268s	1.0/170 1% fury	enduring_torment, monster_rising, egg_sac(3), authority_of_radiant_power, Ascendance_Vers(10), flask_of_alchemical_chaos_mastery
> 2:56.014	J	immolation_aura[default]	Yokaidh	18.0/170 11% fury	enduring_torment, monster_rising, egg_sac(3), authority_of_radiant_power, Ascendance_Vers(10), flask_of_alchemical_chaos_mastery

I sent the ticket to Raidbots with attached the screenshot of your previous message and the Sim I used to open this ticket. I will update you once I have a reply to the case

**Comment by Hekili** [comment]
Closing due to age.  I'll keep watching for sim updates.
